Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'http://rubygems.org'
 
-ruby '2.5.3'
+ruby '~> 2.5.0'
```

Comment:
I don't want to compile _every_ patch of ruby for every different project, so just moved this to any `2.5.x`

Reply:
I didn’t know this worked!